rdkit/SimDivFilters/SimilarityPickers.py

```diff
@@ -110,8 +112,10 @@ def __init__(self, numToPick=10, probeFps=None, dataSet=None,
     self.simMetric = simMetric
     self._picks = None
 
-  def MakePicks(self, force=0):
+  def MakePicks(self, force=False, silent=True):
     if self._picks is not None and not force:
+      if not silent:
+        print('Use force to re-make picks')
```

Comment:
I don't like this at all. If there's going to be a warning printed, it should be a warning using the RDLogger code.